### ISSUE TITLE
[IMPROVEMENTS]: Display Boom Blaster ammo amount

### ISSUE THREAD

**Issue by Z0SC** [typical-developers/oaklands-feedback#753]
### Roblox Username

Z3SC

### What do you think can have quality-of-life improvements?

Boom Blaster tool

### How do you think things can be improved?

I think the boom blaster should have a surface gui on the gun OR screen gui that displays the amount of ammo it has left.
This would just display a number and NOT the type of ammo that its holding, it is up to the player to organize their ammo.

This is an image I found on google of a display example:
![image](https://github.com/user-attachments/assets/e71a79ba-a740-4c4a-a3c0-087bf7c9a6e1)

**Comment by Imjustjokingwithya** [comment]
+1

**Comment by AresWalker8** [comment]
I agree with the concept but it would be nice if it showed an icon next to the ammo count displaying the current explosive that will be launched. Doesn't need to show how much of each item just display that will be fired out as well as total ammo.
